pallets/amm/src/mock.rs

```diff
@@ -154,13 +155,24 @@ impl orml_currencies::Config for Test {
 
 parameter_types! {
     pub const AMMPalletId: PalletId = PalletId(*b"par/ammp");
+    pub const AllowPermissionlessPoolCreation: bool = true;
+      pub const AllowPermissionedPoolCreation: bool = false;
```

Comment:
To avoid confusions :)
```suggestion
      pub const ForbidPermissionlessPoolCreation: bool = false;
```